manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
```

Comment:
Change the namespace to Microsoft.OSTCExtensions.

The namespace Microsoft.Azure.Extensions requires the creation of additional configuration data before it can be run through CI.  I would prefer to defer that work for now.